packages/console/src/pages/SignInExperience/index.tsx

```diff
@@ -156,7 +167,7 @@ const SignInExperience = () => {
           )}
         </Card>
       </div>
-      {formData.id && <Preview signInExperience={previewConfigs} />}
+      {formData.id && <Preview signInExperience={previewConfigs} languageTags={languages} />}
```

Comment:
is no need to pass this prop? the component can `useUiLanguage` by its own.

Reply:
updated